### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by mikesch803**
1. fix form validation
2. fix img upload

### CODE REVIEW

**Review by dev-prabhat — APPROVED** [review]
Looks good to me 
Good to merge

File: src/components/signup-form/SignupForm.jsx [hunk 1]
```diff
@@ -1,109 +1,146 @@
...
-  },[token])
-  
+  }, [token, formDetail, formValidation]);
+
   const signupHandler = (e) => {
     e.preventDefault(e);
-    dispatch(signup(formDetail))
-  }
-
+    formValidation(
```
Comment (kgajriaAgarwal) [inline src/components/signup-form/SignupForm.jsx:33]: suggestion: You can make validation regex for the validations

**Review by kgajriaAgarwal — APPROVED** [review]
code is clean and systematic.
can be merged!

**Review by VLeads — APPROVED** [review]
Looks good. You can merge it !!

**Pull request merged**